app/controllers/tickets_controller.rb

```
@@ -1,51 +1,164 @@
+  # Only allow a list of trusted parameters through
+  def ticket_params
+    params.require(:ticket).permit(
+      :title, :description, :ticket_type, :status, :urgency, :priority, :impact,
+      :assignee_id, :team_id, :category, :caller_name, :caller_surname, :caller_email,
+      :caller_phone, :customer, :source, :reported_at, :requester_id
+    ).tap do |ticket_params|
+      # Ensure required fields are present
+      required_fields = [
+        :title, :description, :ticket_type, :status, :urgency, :priority, :impact,
+        :team_id, :category, :caller_name, :caller_surname, :caller_email, :caller_phone,
+        :customer, :source, :reported_at
```

rubocop (1.69.1)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)